[APIS-919] 11.2 Driver Conformance Test - ADO.NET

hwany7seo opened the pull request:
http://jira.cubrid.org/browse/APIS-919

Purpose
for 11.1.0.0001 Release.

Implementation
- update version (11.1.0.0001)
- update CCI dirver (11.1.0)
- add script for get cci driver library
- modify ado.net driver for count type (int -> bigint)
- added charset in 'T_CCI_COL_INFO ' struct  (sync to CCI Driver)
    (issue : convert error in reader(string -> string[], null error in DataAdapter.Fill)
- update testcase for count type (int -> bigint)
- update testcase for user schema
- update testcase for oid value (11.2 - game oid is '@4481|1|0')
- modify testcase due to default value(DONT_REUSE_OID) change in server 11.2
- modify the wrong testcase (GetSchema(Views).Rows.Count Value)
- added create function(Procedure)  code for testcase (GetSchema_NameAndFilter_Test)

Remarks
N/A

hwany7seo committed:
[APIS-919] 11.2 Driver Conformance Test - ADO.NET

- update version (11.1.0.0001)
- update CCI dirver (11.1.0)
- add script for get cci driver library

hwany7seo committed:
- added gitignore

hwany7seo committed:
- modify ado.net driver for count type (int -> bigint)
- update testcase for count type (int -> bigint)
- update testcase for user schema

kisoo-han commented on Installer/nsis/Release Notes.txt:
```diff
@@ -1,10 +1,10 @@
 Release Notes
 ================================================================================
-January 2021 - Update for release 11.0.0
+January 2021 - Update for release 11.1.0
```
"May 2022" not January 2021

kisoo-han commented on Installer/nsis/Release Notes.txt:
```diff
@@ -1,10 +1,10 @@
-Version: 11.0.0.0001
-CUBRID engine compatible: 11.0.0
+Version: 11.1.0.0001
+CUBRID engine compatible: 11.1.0
```
CUBRID engine compatible: 11.2

hwany7seo replied:
very thanks.

hwany7seo committed:
update testcase for user schema

hwany7seo committed:
update testcase for oid value (11.2 - game oid is '@4481|1|0')

hwany7seo committed:
added create funtion and modify 'user schema' case in testcase (GetSchema_NameAndFilter_Test)

hwany7seo committed:
- added in T_CCI_COL_INFO (sync to CCI Driver)
(issue : convert error in reader(string -> string[], null error in DataAdapter.Fill)

hwany7seo committed:
update release note for engine and release data

hwany7seo committed:
delete unuse files

hwany7seo committed:
- modify testcase due to default value(DONT_REUSE_OID) change in server 11.2
- modify the wrong testcase (GetSchema(Views).Rows.Count Value)

hwany7seo committed:
delete unuse line in script
